addons/web_editor/controllers/main.py

```diff
@@ -16,7 +16,7 @@
 from odoo import http, tools, _, SUPERUSER_ID
 from odoo.addons.http_routing.models.ir_http import slug, unslug
 from odoo.exceptions import UserError
-from odoo.modules.module import get_module_path, get_resource_path
+from odoo.modules import get_module_path, get_resource_path, module
```

Comment:
Why not just add `addons_manifest` to the existing list of imports?

Reply:
Because some tests change the `addons_manifest` reference and if you use the module import reference then there are bugs because the wrong reference is used.